types/src/front/membership_invitation.ts

```diff
@@ -16,7 +15,7 @@ export type MembershipInvitationType = {
 // Types for the invite form in Poke.
 
 export const InviteMemberFormSchema = t.type({
-  email: NonEmptyString,
+  email: t.string,
   role: ActiveRoleSchema,
```

Comment:
Heads up: this wasn't introduced by #5406 dunno if you wanna change it to `t.string`